### DISCUSSION TITLE
some kind a synced chart library between my devices running lazer

### DISCUSSION THREAD

**Discussion by DaniFluffyLab** [ppy/osu#28323] [Ideas] [answered]
A feature i think it will be preeetty useful: a synced chart library between devices.
I've playing osu on my PC and tablet, and, when i open osu! in my tablet, its reaaally boring have to re-search for charts i've already downloaded in my PC.

Sooo let's supposed im logged in my osu! account in PC and download some maps to play.
These downloads will be recorded in my account, and when i open osu! on my tablet, phone, or other PC, these downloads will be showed in my library, with a button to click and download the chart in this device (similar as the update button). If i wanna remove this map from my synced library, i right-clicked and remove from my account.

Even if its was a osu!supporter feature, i think maybe its a good idea, but what you guys think? makes sense some like this?

~non native english speaker, so sowwy for typos :p~

**Comment by BlockHead007** [comment]
#27744 or #27376

**Reply by DaniFluffyLab** [reply, marked as answer]
oooohh... i didnt find these question before i post, sorry for this hehe

CrazyVito11 (#27744) has a preety neat solution, i'll try this later (despite i dont think it will work... android doesnt allow apps to access /data files, so maybe Syncthing will not see the osu files to sync)

and yuuiko (#22073) address exactly what i was thinking...
tnx for the response btw!
